app/assets/javascripts/discourse/app/components/software-update-prompt.js

```diff
@@ -14,9 +16,13 @@ export default Component.extend({
     return getURL("/");
   },
 
+  isHidden: not("showPrompt"),
```

Comment:
I think `isHidden` can be removed completely by changing `attributeBindings` to `["showPrompt::aria-hidden"]`